modules/fee-market/src/lib.rs

```diff
@@ -107,6 +108,17 @@ pub mod pallet {
 		UpdateAssignedRelayersNumber(u32),
 		/// Slash report
 		FeeMarketSlash(SlashReport<T::AccountId, T::BlockNumber, BalanceOf<T, I>>),
+		/// Create new order. \[lane_id, message_nonce, assigned_relayers, order_fee,
```

Comment:
/// Create new order. \[lane_id, message_nonce, order_fee, assigned_relayers

Reply:
No, you need to unescape the symbol.

Reply:
done.